### DISCUSSION TITLE
How can I integrate testing in Nuxt.js with the "@sidebase/nuxt-auth" module?

### DISCUSSION THREAD

**Discussion by Ray0907** [nuxt/nuxt#25655] [Questions]
### Environment

------------------------------
- Operating System: Darwin
- Node Version:     v18.19.0
- Nuxt Version:     3.10.0
- CLI Version:      3.10.0
- Nitro Version:    2.8.1
- Package Manager:  yarn@1.22.19
- Builder:          -
- User Config:      runtimeConfig, app, vite, nitro, modules, auth, i18n, dayjs, routeRules, security
- Runtime Modules:  @sidebase/nuxt-auth@0.5.0, @nuxtjs/i18n@8.0.1, dayjs-nuxt@2.1.9, formidable@3.5.1, nuxt-security@0.14.4, @nuxt/test-utils/module@3.11.0
- Build Modules:    -
------------------------------

### Reproduction

i do not know how to reproduce

### Describe the bug

I keep encountering the same error message when I run "vitest run," even though I have already verified that the URL is correct.
[nuxt] error caught during app initialization FetchError: [GET] "http://localhost:3000/api/auth/session?callbackUrl=http://localhost:3000/": <no response> Fetch to "http://localhost:3000/api/auth/session?callbackUrl=http://localhost:3000/" failed. Error: connect ECONNREFUSED 127.0.0.1:3000

**Comment by yuelongh** [comment]
I am having the same issue. 
There is a similar thread here but since it is mocking global $fetch I seem to be getting more errors from Nuxt/other modules internal use of $fetch 

https://github.com/nuxt/test-utils/issues/547
